### DISCUSSION TITLE
Executing multiple files at the same time with simulation. And Using the following way

### DISCUSSION THREAD

**Discussion by shizhiman** [pyswmm/pyswmm#352] [Q&A]
I want to concurrent emulation, executing multiple files at the same time with simulation. And Using the following way：

sim = Simulation(file)
for step in sim:
  test = 1
sim.close()

How do I do it or with what package？

**Comment by shizhiman** [comment]
I have solved the problem

**Reply by bemcdonnell** [reply]
@shizhiman Can you share the code that you used? I would like to add this to a tutorial if you don't mind.
